packages/clerk-js/rspack.config.js

```diff
@@ -356,54 +387,66 @@ const prodConfig = ({ mode, env, analysis }) => {
     entryForVariant(variants.clerkHeadlessBrowser),
     common({ mode }),
     commonForProd(),
-    commonForProdChunked(),
+    commonForProdChunked({ targets: packageJSON.browserslist }),
     // externalsForHeadless(),
   );
 
-  const clerkEsm = merge(entryForVariant(variants.clerk), common({ mode }), commonForProd(), commonForProdBundled(), {
-    experiments: {
-      outputModule: true,
-    },
-    output: {
-      filename: '[name].mjs',
-      libraryTarget: 'module',
-    },
-    plugins: [
-      // Include the lazy chunks in the bundle as well
-      // so that the final bundle can be imported and bundled again
-      // by a different bundler, eg the webpack instance used by react-scripts
-      new rspack.optimize.LimitChunkCountPlugin({
-        maxChunks: 1,
-      }),
-    ],
-    optimization: {
-      splitChunks: false,
+  const clerkEsm = merge(
```

Comment:
none of this was actually changed, it's just a formatting thing

Reply:
Where does the formatting change come from then? Your personal prettier settings? If I open the file in my editor ESLint doesn't want to format the file.

Ideally such unintended formatting changes don't make it into the PR as it's quite possible that another PR will undo them later again.

Reply:
Previously Prettier kept everything on one line since it was below our max line length threshold. Since the line is longer (due to the inclusion of the `{ targets: packageJSON.browserslist }` argument), it splits the arguments to `merge` into separate lines. This confuses the GitHub diff formatter into thinking the lines have changed due to how the final argument to `merge` is now indented. So this is Prettier working as expected! I just wanted to make sure folks weren't struggling to find the difference between the two :)